damus.xcodeproj/project.pbxproj

```diff
@@ -738,11 +739,12 @@
 			};
 			buildConfigurationList = 4CE6DEDE27F7A08100C66700 /* Build configuration list for PBXProject "damus" */;
 			compatibilityVersion = "Xcode 13.0";
-			developmentRegion = en;
```

Comment:
Using `en` as the development locale makes it ambiguous. If we ever introduce strings like `color`/`colour` or `favorite`/`favourite`, it'll be inconsistent. Since Apple intentionally uses `en-US`, we'll use `en-US` for Damus as well.